cms/templates/cms/home_page.html

```diff
@@ -68,180 +74,169 @@
         {% endif %}
         <div class="banner-wrapper-content">
           <h1>
-            MITx MicroMasters Programs: Bringing MIT to you
+            MITx MicroMasters Programs: Master Your Future
```

Comment:
According to [invisio design](https://projects.invisionapp.com/share/YFZ4N7KMWT3#/screens/435513872_Homepage_V2), the font-weight of `MITx MicroMasters Programs` is bigger than `Master Your Future`.
And also the `banner-wrapper-content` div is wider.

Reply:
Good eye. We already have approval on the consistent font sizing in the PR build, so let's keep this as is.